onto_spread_ed/templates/admin/dashboard.html

```diff
@@ -7,7 +7,7 @@
 
     <div class="row align-stretch">
         <div class="col-sm-6 col-md-4 col-lg-3 m-2">
-            <a class="card bg-light" href="/admin/release"
+            <a class="card bg-light" href="{{ config['URL_PREFIX'] }}/admin/release"
```

Comment:
Is there a reason you didn't use `url_for` here?

Reply:
Initially when we were changing the prefixes, all the href in html files were changed in this following way itself and it worked and for those which didnt work properly we added the url_for as part of our solution in the next phase.
So most of the hrefs are configured similarly. If we have to revert, then we would require to change those accordingly.

Reply:
Using `url_for` is cleaner than manually constructing the URL with the value from the config. Please use `url_for` where possible.